css/header.css

```diff
@@ -0,0 +1,138 @@
+.header {
+  /* border: 3px solid black; */
+  display: flex;
+  /* padding: 62px 50px 0px 50px; */
+  font-family: "Source Code Pro";
+  font-style: normal;
+  line-height: normal;
+  background-color: #f4f4f4;
+  height: 100vh;
+	position: relative;
+}
+.header__row {
+  display: flex;
+	/* position: relative; */
+	/* border: 1px solid #000; */
+	height: 100%;
+	padding: 60px 0px;
+}
+.header__about {
+  width: 50%;
+  margin: 0 auto 0 0;
+}
+.header__top {
+	margin: 0px 0px 85px 0px;
+}
+.header__nav {
+}
+.nav {
+  line-height: normal;
+  font-size: 22px;
+  font-weight: 250;
+}
+.nav__list {
+  display: flex;
+  /* border: 1px solid black; */
+	/* position: relative; */
+	/* padding: 0px 0px 0px 80px; */
+}
+
+.nav__link-btn {
+	display: inline-block;
+	width: 32px;
+	height: 32px;
+	background-color: black;
+	border-radius: 50%;
+	/* margin: 0px 80px 0px 0px; */
+	/* position: absolute; */
+	top: 50%;
+	left: 50%;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	color: white;
+}
+
+.nav__item:not(:last-child) {
+  margin: 0 26px 0 0;
+}
+.nav__item:not(:last-child):not(:first-child)::after {
+  content: '';	
+	width: 3px;
+	height: 3px;
+	background-color: #000;
+	display: inline-block;	
+	position: relative;
+	top: -15%;
+	left: 10%;
+}
+.nav__link {
+  color: #000;
+}
+
+.header__article {
+  font-size: 60px;
+  font-weight: 500;
+  letter-spacing: -0.5px;
+  margin: 0 30px 145px 0;
+  position: relative;
+}
+.header__article::after {
+  content: "";
+  position: absolute;
+  display: inline-block;
+  background: url("../img/first-screen/bad-lines.png");
+  width: 76px;
+  height: 74px;
+  bottom: -60%;
+  left: 0;
+}
+.header__describe {
+  font-size: 16px;
+  font-weight: 400;
+  max-width: 375px;
+  margin: 0 0 111px 0;
+}
+.header__button {
+}
+.button__main {
+}
+.header__image {
+  position: relative;
```

Comment:
this css selector has 3 `position` rules
only 1 will work, so you have to delete all useless rules
![image](https://github.com/zahurnyi/school/assets/96112534/2feb3610-2e00-422d-a2ca-3ef9d62c0501)